noir-projects/aztec-nr/aztec/src/encrypted_logs/payload.nr

```diff
@@ -1,326 +0,0 @@
-use dep::protocol_types::{
```

Comment:
This has been moved to default_aes128/note.nr (and event.nr and partial_note.nr). partial_note.nr will soon die. note.nr will soon have the randomness optimisation, so is kept distinct from event.nr paths, until we know how to modularise things without prematurely causing spaghetti.